validation_pipeline.ipynb

```diff
@@ -7,7 +7,7 @@
         "colab_type": "text"
```

Comment:
We do joins because our current implementation only calculates metrics with ungrouped data. We're changing this here: https://github.com/tnc-br/ddf_common/pull/30

The input test data is grouped, unfortunately, so we "reverse" that. I will change this to the grouped version once that is checked in.

Reply:
<p>I still don't think I understand what you mean here. How is it possible to reverse aggregation?</p><p> </p><p>Also, it seems like this first line is a no-op. Is that intentional?</p><p><code></code><code></p><p>&nbsp;test_dataset = pd.concat([real, fake_sample], axis=1, join='outer')</p><p>&nbsp;test_dataset = real.append(fake_sample, ignore_index=True)</p><p></code><code></code></p><p> </p>

Reply:
You take the unique code, long, lat codings and join them with the ungrouped dataset. This will result in a dataset that has the original 5 samples but now they have the aggregate mean and variance columns.

Reply:
AI: Delete the first line